docs/imviz/displayimages.rst

```diff
@@ -0,0 +1,85 @@
+.. _imviz-display-images:
+
+*****************
+Displaying Images
+*****************
+
+Imviz uses image viewers to visualize data from supported formats.
+
+Selecting Data Set
+==================
+
+.. seealso::
+
+    :ref:`Selecting Data Set<cubeviz-selecting-data>`
+        Documentation on selecting data sets in the Jdaviz viewers.
+
+
+Adding New Viewers
+==================
+
+In the toolbar towards the top of the UI, there is a :guilabel:`picture with +` icon. By
+clicking this icon, you can add new viewers to the application. You can then select from the data
+that has been loaded into the application to be visualized in these additional viewers.
+You can then utilize some of the Imviz-specific features, like Linked Pan/Zoom.
+
+Defining Spatial Regions
+========================
+
+Spatial regions can be created the same way as in Cubeviz.
+
+.. seealso::
+
+    :ref:`Defining Spatial Regions <spatial-regions>`
+        Documentation on defining spatial regions in an image viewer.
+
+Blinking
+========
+
+Blinking is an Imviz-specific functionality that allows a user to quickly switch
+between viewing two images. This can be done by selecting the :guilabel:`hammer and screwdriver`
```

Comment:
Is it two only, or two or more? Might be worth saying whether the images to blink between need to be selected in the data menu or not.

Reply:
@pllim ? I was not able to blink between more than two images, not sure if that is user error or not.